src/main/java/mate/academy/dao/MovieDao.java

```diff
@@ -10,4 +10,8 @@
     Optional<Movie> get(Long id);
 
     List<Movie> getAll();
+
+    boolean update(Movie movie);
+
+    boolean delete(Long id);
```

Comment:
The return type of the `delete` method should be `boolean` instead of `Boolean` to avoid unnecessary boxing.